### PR TITLE
fix scrape path &  scheme for crdb & jsonnet

### PR DESCRIPTION
**Pull request by charlie-pisuraj**

### CODE REVIEW

File: cloud_build.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "steps": [
```
Comment (steeling) [inline cloud_build.json]: what is this file for? is it supposed to be part of this change?

**Comment by charlie-pisuraj** [comment]
@steeling, I've disabled scrapping annotations for grpc backend and http backend since there is no endpoint to hit, please re-review

File: build/deploy/grpc-backend.libsonnet [hunk 2]
```diff
@@ -6,7 +6,7 @@ local volumes = import 'volumes.libsonnet';
     service: base.Service(metadata, 'grpc-backend') {
       app:: 'grpc-backend',
       port:: metadata.backend.port,
-      enable_monitoring:: true,
+      enable_monitoring:: false,
```
Comment (steeling) [inline build/deploy/grpc-backend.libsonnet]: you can just omit these since it defaults to false in the base

Comment (charlie-pisuraj) [reply build/deploy/grpc-backend.libsonnet]: I really prefer making it explicitly clear that monitoring is off, an when the time comes when we want to turn it on we know where it goes, more importantly the user of this repo will know by just searching "monitoring"

**Pull request merged**